README.md

```diff
@@ -13,10 +13,17 @@ In addition to tests, this package creates the model `int_all_dag_relationships`
 
  This package is in its early stages! It's very likely that you could encounter bugs, and functionality will be changing quickly as we gather feedback from end users. Please do not hesitate to create new issues in this repo for bug reports and/or feature requests, and we appreciate your patience as we continue to enhance this package! 
 
-
 ## Installation Instructions
 Check [dbt Hub](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/) for the latest installation instructions, or [read the docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
 
+## How To Use
+This package will:
+1. Parse your [graph](https://docs.getdbt.com/reference/dbt-jinja-functions/graph) object and write it into your warehouse as a series of models (see [models/marts/core](https://github.com/dbt-labs/dbt-project-evaluator/tree/main/models/marts/core))
+2. Create another series of models that each represent one type of misalignment in your project (see [models/marts/dag](https://github.com/dbt-labs/dbt-project-evaluator/tree/main/models/marts/dag), [models/marts/documentation](https://github.com/dbt-labs/dbt-project-evaluator/tree/main/models/marts/documentation), [models/marts/structure](https://github.com/dbt-labs/dbt-project-evaluator/tree/main/models/marts/structure), and [models/marts/tests](https://github.com/dbt-labs/dbt-project-evaluator/tree/main/models/marts/tests))
```

Comment:
wondering if it would be clearer to reference the README (e.g. "below you can find a full list of each misalignment and its accompanying model") instead of linking to folders in the repo

Reply:
Agreed - the README is getting pretty massive.  Maybe we could use toggle lists to hide some information, like installation instructions, similar to here: https://gist.github.com/christineberger/2d00a2f846b1b49359fde5c2f8f354fe

Reply:
done and done :)